### DISCUSSION TITLE
Migrating v7 to v8. UseFlexLayout hook equivalent in v8.

### DISCUSSION THREAD

**Discussion by Cicada95** [TanStack/table#4442] [General]
Hi, guys I'm migrating react-table to new v8. In v7 there was this custom hook called useFlexLayout, what would be the equivalent of it in new react-table v8.

**Comment by konsalex** [comment]
Not a hook per-se but a cookbook of a spike I did:
https://codesandbox.io/s/react-table-8-full-width-5id1g6?file=/src/main.tsx

Basically used a display `grid`, and a `<div />` table approach. Not sure though if it helps you, but the space allocation (if you see the last column) works smooth

https://user-images.githubusercontent.com/12672541/221572235-189e302b-19d7-4982-a0c1-9367e850faca.mp4
